include/audio/soundfont.h

```diff
@@ -0,0 +1,101 @@
+#ifndef AUDIO_SOUNDFONT_H
+#define AUDIO_SOUNDFONT_H
+
+#include "PR/ultratypes.h"
+
+struct EnvelopePoint;
+
+typedef struct AdpcmLoop {
+    /* 0x00 */ u32 start;
+    /* 0x04 */ u32 loopEnd; // numSamples into the sample where the loop ends
+    /* 0x08 */ u32 count; // The number of times the loop is played before the sound completes. Setting count to -1 indicates that the loop should play indefinitely.
+    /* 0x0C */ u32 sampleEnd; // total number of s16-samples in the 
```

Comment:
in the what?

Reply:
Oops. It should just be in the entire sample audio clip